### DISCUSSION TITLE
Guidance on Selecting and Combining DPA2 Models for Water-Metal Interface Structures

### DISCUSSION THREAD

**Discussion by poyaii** [deepmodeling/deepmd-kit#4061] [Q&A] [answered]
Hello. 
Thank you for providing the DPA2 models. If I want to use these models for a water-metal interface structure, which model would be more suitable? Is it possible to use multiple models, for example, starting with H2O_H2O-PD and then Alloy? I would appreciate your guidance.   

Best
Poya

**Comment by wanghan-iapcm** [comment, marked as answer]
The accuracy of the finetuned model is not very sensitive to the choice of fitting head. please refer to Figs. S2 of https://arxiv.org/abs/2312.15492
